### PR TITLE
und-737 form changes

### PR DESCRIPTION
**Pull request by Ashish-srij**
firm issues fixed

### CODE REVIEW

File: stories/Components/Forms/Textpanel/FormTypes.jsx [hunk 1]
```diff
@@ -145,35 +148,37 @@ export const FormTypes = ({
...
-                      placeholder={placeholderemail}
-                      required={required}
-                      mode={mode}
-                    />
-                  </div>
-                  <div className="cell">
-                    <button type="submit" className="button button-primary">
-                      {button}
-                    </button>
+        <div className='stacked-form'>
```
Comment (shishirphadke) [inline stories/Components/Forms/Textpanel/FormTypes.jsx:151]: @Ashish-srij Remove `stacked-form` class if not being used as it adds to unnecessary extra divs. Else use it in the existing div structure.

File: stories/Components/Forms/Textpanel/FormTypes.jsx [hunk 2]
```diff
@@ -17,43 +17,46 @@ export const FormTypes = ({
...
-              <div className="cell">
-                <Inputcomponent
-                  element={element}
-                  type={type}
-                  placeholder={placeholderemail}
-                  required={required}
-                  mode={mode}
-                />
-              </div>
+        <div className='text-panel'>
```
[the remark this answers is not in the document]
Comment (Ashish-srij) [reply stories/Components/Forms/Textpanel/FormTypes.jsx]: @shishirphadke Done.

File: stories/assets/scss/_mixins.scss [hunk 3]
```diff
@@ -420,9 +420,16 @@
 
 %carousel_arrows{
   display: none;
-  
+
   @media (pointer: coarse) and (hover: none) and (min-width: 48em) {
     display: block;
   }
 }
+%verticl_center-mb{
```
[the remark this answers is not in the document]
Comment (Ashish-srij) [reply stories/assets/scss/_mixins.scss]: @shishirphadke Client requirement changed so I have removed the "verticl_center-mb".

File: stories/Components/Forms/Textpanel/FormTypes.scss [hunk 4]
```diff
@@ -1,8 +1,9 @@
-@import "../../../assets/scss/variables";
-@import "../../../assets/scss/mixins";
+@import '../../../assets/scss/variables';
+@import '../../../assets/scss/mixins';
 
 .form-wrapper {
     .text-panel{
+        @extend %verticl_center-mb;
```
[the remark this answers is not in the document]
Comment (Ashish-srij) [reply stories/Components/Forms/Textpanel/FormTypes.scss]: @shishirphadke Client requirement changed so I have removed the "verticl_center-mb".

File: stories/Components/Forms/Textpanel/FormTypes.scss [hunk 5]
```diff
@@ -59,10 +62,11 @@
         }
         .button {
                 margin-top: $spacing-16;
-            
+
         }
     }
     .stacked-form {
+        @extend %verticl_center-mb;
```
[the remark this answers is not in the document]
Comment (Ashish-srij) [reply stories/Components/Forms/Textpanel/FormTypes.scss]: @shishirphadke Client requirement changed so I have removed the "verticl_center-mb".

File: stories/Components/Forms/Textpanel/FormTypes.jsx (hunk 1 above)
Comment (Ashish-srij) [reply stories/Components/Forms/Textpanel/FormTypes.jsx]: @shishirphadke Div removed and class moved to bottom div

**Pull request merged**